packages/balancer/src/Balancer.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  type TransactionFilter,
+  type SwapActionParams,
+  compressJson,
+} from '@rabbitholegg/questdk'
+import { type Address } from 'viem'
+import { BALANCER_ABI } from './abi'
+import { CHAIN_ID_ARRAY, VAULT_CONTRACT } from './constants'
+import { buildAmountQuery, buildPathQuery } from './utils'
+import { CHAIN_TO_TOKENS } from './token-addresses'
+
+export const swap = async (
+  swap: SwapActionParams,
+): Promise<TransactionFilter> => {
+  const {
+    chainId,
+    contractAddress,
+    tokenIn,
+    tokenOut,
+    amountIn,
+    amountOut,
+    recipient,
+  } = swap
+
+  return compressJson({
+    chainId: chainId,
+    to: contractAddress ?? VAULT_CONTRACT,
```

Comment:
Can you do me a favor and add the `0x5ff137d4b0fdcd49dca30c7cf57e578a026d2789` entry point contract as an array for the `$to` here - patch wallet wants to use this plug-in to test account abstraction with plugins.